### PR TITLE
MFTF: Replace repetitive actions with Action Groups in AdminUpdateStoreGroupAcceptAlertAndVerifyStoreViewFormTest, AdminUpdateStoreGroupAndVerifyStoreViewFormTest and AdminUpdateStoreViewTest

### PR DESCRIPTION
**Pull request by kate-kyzyma**
### Description (*)
AdminUpdateStoreGroupAcceptAlertAndVerifyStoreViewFormTest, AdminUpdateStoreGroupAndVerifyStoreViewFormTest and AdminUpdateStoreViewTest are refactored according to the best practices followed by MFTF.

### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)
Run tests in the local environment and verified the tests run successfully.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by sidolov — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-8281](https://jira.corp.magento.com/browse/ENGCOM-8281) has been created to process this Pull Request
:eight_spoked_asterisk: @sidolov, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

File: app/code/Magento/Store/Test/Mftf/ActionGroup/AdminOpenFirstRowInStoresGridActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminOpenFirstRowInStoresGridActionGroup">
+
```
Comment (lbajsarowicz) [inline app/code/Magento/Store/Test/Mftf/ActionGroup/AdminOpenFirstRowInStoresGridActionGroup.xml:12]: @kate-kyzyma Could you lint the XML files and remove the empty (redundant) spaces?

File: app/code/Magento/Store/Test/Mftf/ActionGroup/AdminOpenFirstRowInStoresGridActionGroup.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        <click selector="{{AdminStoresGridSection.firstRow}}" stepKey="clickFirstRow"/>
+        <waitForPageLoad stepKey="AdminSystemStoreGroupPageToOpen"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/Store/Test/Mftf/ActionGroup/AdminOpenFirstRowInStoresGridActionGroup.xml:14]: Step key should start lowercase
```suggestion
        <waitForPageLoad stepKey="adminSystemStoreGroupPageToOpen"/>
```

File: app/code/Magento/Store/Test/Mftf/ActionGroup/AdminOpenStoreInFirstRowInStoresGridActionGroup.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminOpenStoreInFirstRowInStoresGridActionGroup">
+
```
Comment (lbajsarowicz) [inline app/code/Magento/Store/Test/Mftf/ActionGroup/AdminOpenStoreInFirstRowInStoresGridActionGroup.xml:12]: Remove empty space

File: app/code/Magento/Store/Test/Mftf/Test/AdminUpdateStoreGroupAcceptAlertAndVerifyStoreViewFormTest.xml [hunk 4]
```diff
@@ -51,8 +51,8 @@
         <actionGroup ref="AssertStoreGroupInGridActionGroup" stepKey="openCreatedStoreGroupInGrid">
             <argument name="storeGroupName" value="{{staticStoreGroup.name}}"/>
         </actionGroup>
-        <click selector="{{AdminStoresGridSection.firstRow}}" stepKey="clickFirstRow"/>
-        <waitForPageLoad stepKey="AdminSystemStoreGroupPageToOpen"/>
+        <actionGroup ref="AdminOpenFirstRowInStoresGridActionGroup" stepKey="clickFirstRow"/>
+
         <!--Update created Store group as per requirement and accept alert message-->
```
Comment (lbajsarowicz) [inline app/code/Magento/Store/Test/Mftf/Test/AdminUpdateStoreGroupAcceptAlertAndVerifyStoreViewFormTest.xml:56]: As comments are displayed just like other steps - it's recommended to continuously remove the comments, as these do not have any handler to remove them from the listing of steps performed :-(

**Pull request merged**